dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/VisitorSpirv.java

```diff
@@ -0,0 +1,15 @@
+package com.dat3m.dartagnan.parsers.program.visitors;
+
+import com.dat3m.dartagnan.configuration.Arch;
+import com.dat3m.dartagnan.expression.Expression;
+import com.dat3m.dartagnan.parsers.SpirvBaseVisitor;
+import com.dat3m.dartagnan.parsers.program.utils.ProgramBuilder;
+import com.dat3m.dartagnan.program.Program;
+
+public class VisitorSpirv extends SpirvBaseVisitor<Expression> {
+
+    private final ProgramBuilder programBuilder = ProgramBuilder.forArch(Program.SourceLanguage.LITMUS, Arch.TSO);
```

Comment:
This does not make sense.

We should probably add a new entry to `Program.SourceLanguage` and set the target arch to `Arch.VULKAN`

Reply:
Oops, fixed